Tests/PostgresNIOTests/New/Extensions/ConnectionAction+TestUtils.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 12.01.21.
+//
+
+import class Foundation.JSONEncoder
```

Comment:
Ditch Xcode template headers.
```suggestion
import class Foundation.JSONEncoder
```